app/components/Form/ApplicationForm.tsx

```diff
@@ -177,12 +179,52 @@ const ApplicationForm: React.FC<Props> = ({
       ) : (
         <Button variant="primary">Submit</Button>
       )}
-      {/* // Return to this save button later, will likely require a hacky solution to work
-      // nice with RJSF
-      <Button variant="secondary" style={{ marginLeft: '20px' }}>
-        Save
-      </Button> */}
-    </FormBase>
+    </>
+  );
+
+  return (
+    <>
+      {isCalculatedPage ? (
```

Comment:
@VB-at-Bis @AntBush what do you guys think about this? I wasn't sure if there is a cleaner way to go about using the `CalculatedForm` renderer. This way we only use it on the pages that use the calculations but it feels a bit messy somehow.

Reply:
It looks good for me :) .. not messy at all

Reply:
I think this way for one or two it's fine, but we'll definitely have to think of a different way to do this if we have many custom forms we want to display.

Reply:
Did you check how calculated fields are handled in CIIP and CIF. Adding some section-specific branching in the `ApplicationForm` page is not a maintainable solution. The rjsf way of doing this is to define custom fields for those items that require calculations